### DISCUSSION TITLE
显示列表有一点小瑕疵

### DISCUSSION THREAD

**Discussion by 1175347013** [AdNotFound/EhViewer-Neko#15] [Q&A] [answered]
列表里，评分和和上传用户会显示重叠，不影响正常使用
型号：一加7Pro/一加8Pro
安卓版本：12/13
分辨率：3120×1440
![Screenshot_2023-11-12-15-50-24-82_3a3ec5e415dc4e14acdcdd730fac53f1.jpg](https://github.com/AdNotFound/EhViewer-Neko/assets/94844907/226e0dc7-ee45-4f45-98fb-77143f596000)

**Comment by AdNotFound** [comment, marked as answer]
![Screenshot_2023-11-12-17-26-07-945_org error ehviewer](https://github.com/AdNotFound/EhViewer-Neko/assets/148407324/3fcfea85-3118-4bca-b8f3-fd4f96815181)
通过修改该设置项到合适大小可以解决这个问题

**Reply by 1175347013** [reply]
OK，谢谢
